certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1557 @@
+    "PubCS.TitleOrder.Api.Models.AddressType": {
+      "type": "string"
+    },
+    "PubCS.TitleOrder.Api.Models.Address": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "type": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.AddressType"
+        },
+        "line1": {
+          "description": "Gets or Sets Line1",
+          "type": "string"
+        },
+        "line2": {
+          "description": "Gets or Sets Line2",
+          "type": "string"
+        },
+        "city": {
+          "description": "Gets or Sets City",
+          "type": "string"
+        },
+        "state": {
+          "description": "Gets or Sets State",
+          "type": "string"
+        },
+        "zip": {
+          "description": "Gets or Sets Zip",
+          "type": "string"
+        },
+        "county": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.County"
+        },
+        "country": {
+          "description": "Gets or Sets Country",
+          "type": "string"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.Tax": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "category": {
+          "description": "Gets or Sets Category",
+          "type": "string"
+        },
+        "payee": {
+          "description": "Gets or Sets Payee",
+          "type": "string"
+        },
+        "amount": {
+          "format": "double",
+          "description": "Gets or Sets Amount",
+          "type": "number"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.Property": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "propertyType": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.PropertyType"
+        },
+        "attributes": {
+          "description": "Gets or Sets Attributes",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "sequence": {
+          "format": "int32",
+          "description": "Gets or Sets Sequence",
+          "type": "integer"
+        },
+        "taxIdentifier": {
+          "description": "Gets or Sets TaxIdentifier",
+          "type": "string"
+        },
+        "address": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.Address"
+        },
+        "parcel": {
+          "description": "Gets or Sets Parcel",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "taxes": {
+          "description": "Gets or Sets Taxes",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.Tax"
+          }
+        },
+        "legalDescription": {
+          "description": "Gets or Sets LegalDescription",
+          "type": "string"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.TitleOrderCommitmentSchedA": {
+      "description": "",
```

Comment:
please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.